lib/reporters/tap.js

```diff
@@ -27,6 +27,15 @@ TapReporter.prototype.start = function () {
 TapReporter.prototype.test = function (test) {
 	var output;
 
+	var directive = '';
+	var passed = test.todo ? 'not ok' : 'ok';
+
+	if (test.todo) {
+		directive = '# TODO';
```

Comment:
If I'm not mistaken, there's no such directive as `# TODO` in TAP spec. For TAP, we could treat `todo` tests just like `skip` and output them the same way.